tests/core/test_cost_calculation.py

```diff
@@ -55,8 +55,10 @@ async def test_basics(self):
         clvm_cost = result.cost
 
         error, npc_list, cost = get_name_puzzle_conditions(program, False)
+        assert error is None
         coin_name = npc_list[0].coin_name
         error, puzzle, solution = get_puzzle_and_solution_for_coin(program, coin_name)
+        assert error is None
```

Comment:
these asserts would fail before this patch, because `program` here is a `SerializedProgram`, and it doesn't automatically de-serialize when passed to `Program.to()`. So an alternative approach would be to de-serialize the program here. However, this is run from mempool cost check, so we should really stick to serialized programs there, for the full node validations